deploy/esignet-with-plugins/install.sh

```
@@ -219,17 +213,11 @@ EOF
       plugin_option="$plugin_option -f $plugin_env_file"
   fi
 
-  extra_env_vars_cm_set=""
-  if kubectl get configmap kafka-config -n "$NS" > /dev/null 2>&1; then
-    extra_env_vars_cm_set="--set extraEnvVarsCM={esignet-softhsm-share,kafka-config}"
-  fi
-
-
   echo Installing esignet-with-plugins
   helm -n $NS install $ESIGNET_SERVICE_NAME mosip/esignet --version $CHART_VERSION  \
     $ENABLE_INSECURE $plugin_option \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)